user/hare/patches/linker-script-execstack.patch

```diff
@@ -0,0 +1,13 @@
+otherwise cbuild flags hare and haredoc binaries as using execstack
```

Comment:
should also upstream this i think, think nobody has just noticed this yet?

Reply:
will do yeah